Dockerfile

```diff
@@ -1,25 +1,32 @@
 ARG DOCKER_ORG=usdotfhwastoldev
 ARG DOCKER_TAG=develop
-ARG GIT_BRANCH=develop
 FROM ${DOCKER_ORG}/carma-base:${DOCKER_TAG} as base_image
 SHELL ["/bin/bash", "-c"]
 
 ARG DEBIAN_FRONTEND="noninteractive"
+ARG VERSION
+ARG VCS_REF
+ARG BUILD_DATE
```

Comment:
I'm not sure on the best practice here, but I'm assuming it would be best to instantiate these with default values. Could we explicitly set `VERSION`, `VCS_REF`, and `BUILD_DATE` to `"NULL"` by default, similar to how they're declared in our other Dockerfiles (such as [carma-platform's](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Dockerfile))

Reply:
I think we need these arguments as they are generated during execution of dockerhub.yml from our actions repo (https://github.com/usdot-fhwa-stol/actions/blob/6051687fa855a6dcddc107f0784fc56ba3aec5d7/.github/workflows/dockerhub.yml#L79). those 3 values are generated and passed through this script (https://github.com/usdot-fhwa-stol/actions/blob/main/docker-build-metadata/action.yml).

Reply:
Sorry if my comment wasn't clear. I agree they are definitely needed. I'm wondering though if you can update them to have a default value of `"NULL"` similar to how they are declared in our other Dockerfiles?

Reply:
Sorry for the misunderstanding, I've updated the default values in our Docker file to 'NULL' as suggested. Thanks! for your help!"